docs/manual/docs/examples/spo/disable-tenant-wide-extension.md

```diff
@@ -0,0 +1,54 @@
+# Disable specified Tenant-wide Extension
+
+Author: [Shantha Kumar T](https://www.ktskumar.com/2020/04/manage-tenant-wide-extensions-using-office-365-cli/)
+
+Tenant Wide Extensions list from the App Catalog helps to manage the activation / deactivation of the tenant wide extensions. The below sample script helps to disable the specifed tenant wide extension based on the id parameter.
+
+Note: TenantWideExtensionDisabled column denotes the extension is enabled or disabled.
+
+
+```powershell tab="PowerShell Core"
+$extensionName = Read-Host "Enter the Extension Name"
+$listName = "Tenant Wide Extensions"
+
+$appcatalogurl = o365 spo tenant appcatalogurl get
+$filterQuery = "Title eq '"+ $extensionName +"'"
+$appitems = o365 spo listitem list --title $listName --webUrl $appcatalogurl --fields "Id,Title" --filter $filterQuery --output json
+$extItems = $appitems.Replace("Id","ExtId") | ConvertFrom-JSON
+
+if($extItems.count -ge 0){
+o365 spo listitem set --listTitle $listName --id $extItems.ExtId --webUrl $appcatalogurl --TenantWideExtensionDisabled "true"; 
```

Comment:
If the command fails, the script continues showing that the extension has been disabled while it's not the case. Before printing the next line, we should check the CLI exit code to verify if disabling the extension actually succeeded